spec/features/cco/defence_request_management_spec.rb

```diff
@@ -36,12 +36,15 @@
 
       specify "can acknowledge the request" do
         cco_user = create :cco_user
-        create :defence_request, :queued, cco_uid: cco_user.uid
+        dr =  create :defence_request, :queued, cco_uid: cco_user.uid
 
         login_with cco_user
         click_button "Acknowledge"
 
         expect(page).to have_content "Defence Request successfully acknowledged"
+
+        click_link "Show"
+        expect(page).to have_content "#{dr.dscc_number}"
```

Comment:
I see your point. Although we wouldn't be doing more that spying on a message send. In that regard, the transition does not _perform_ the work or exercise that functionality - nor should it. It feels the proper place for the test would be in the unit that actually does the work. What do you think?